package.json

```diff
@@ -51,10 +51,10 @@
     "minimist": "^1.2.0",
     "npm-run-all": "^4.1.5",
     "path": "^0.12.7",
-    "prettier": "^1.16.4",
     "plugin-error": "^1.0.1",
+    "prettier": "^1.16.4",
     "ts-node": "^7.0.1",
-    "typescript": "^3.2.2",
+    "typescript": "^3.7.4",
```

Comment:
Did you intend to change the version of TS? Make sure that is what you want as I know different versions of TS have some compatibility issues.

Reply:
No I didn't intend to do this. I'll revert it back.